apps/backoffice-v2/src/domains/workflow-definitions/fetchers.ts

```diff
@@ -27,6 +27,7 @@ export const WorkflowDefinitionConfigSchema = z
     isAssociatedCompanyKybEnabled: z.boolean().default(false),
     isCaseOverviewEnabled: z.boolean().default(false),
     isCaseRiskOverviewEnabled: z.boolean().default(false),
+    isDocumentTrackingEnabled: z.boolean().default(false),
```

Comment:
I think it's for the best that the feature flag would match the UI component it controls.
```suggestion
    isDocumentTrackerEnabled: z.boolean().default(false),
```